.github/workflows/docs-deploy.yml

```diff
@@ -22,5 +20,13 @@ jobs:
         run: |
           pip install mkdocs-material
 
-      - name: Build and Deploy
-        run: mkdocs gh-deploy --force
\ No newline at end of file
+      - name: Configure Git
```

Comment:
Why didn't [they](https://github.com/statisticsnorway/nais-system/blob/main/.github/workflows/publish-gh-pages.yml) have to do that?

Reply:
Not sure, maybe they have different repo settings?

Saw multiple places saying you needed to add credentials
from the material theme:
https://squidfunk.github.io/mkdocs-material/publishing-your-site/
it was one of the things which the previous deploy action said they handled..